client/src/components/ProgrammeRequirementsItem.js

```diff
@@ -3,6 +3,16 @@ import { Flex, Box, Text, Stack } from "@chakra-ui/core";
 import { AiOutlineEdit } from "react-icons/ai";
 
 const ProgrammeRequirementsItem = ({ itemNumber, pointRequirement, points, courseList, onDelete, onEdit, deleteButton }) => {
+    const getPointRequirementString = (pointRequirementValue) => {
+        if (pointRequirementValue === "EXACT") {
+            return "Exactly";
+        } else if (pointRequirementValue === "ATLEAST") {
+            return "At least";
+        } else {
+            return "Up to";
+        }
+    };
+
```

Comment:
You reckon we should have an object for this function?

const pointRequirementMap = {
    "EXACT": "Exactly",
     "ATLEAST": "At least",
}

Then it would just be `pointRequirementMap[pointRequirementValue] || "Up to"`